OpenKore/openkore
# Enable the breakTime plugin

* this might fix https://github.com/OpenKore/openkore/issues/145 
* move plugin to `plugins`
* enable plugin in `sys.txt`
* add documentation to plugin's `README.md` (should this be somehow auto-sync'd with the wiki?)
* add unit tests
* fix a couple of bugs:
  * break times spanning midnight would not trigger if the current time was after midnight but before `stopTime`
  * the plugin would reset the login time if the user had deliberately logged out for a long time (longer than `stopTime`) - it should only extend the time spent logged out, not shorten it
